.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+- package-ecosystem: "docker"
```

Comment:
You need to add open-pull-requests-limit for each package-ecosystem

Reply:
Done. I assume that at most one PR will be opened for `package-ecosystem: docker``, because the go version is the only thing that needs to be updated. But I agree there's no harm to set a limitation here.